ambry-router/src/test/java/com.github.ambry.router/NonBlockingRouterTest.java

```diff
@@ -395,6 +395,50 @@ public void onBlobDeleted(String blobId) {
   }
 
   /**
+   * Test that if a composite blob is deleted, the data chunks are eventually deleted.
+   */
+  @Test
+  public void testCompositeBlobDataChunksDelete() throws Exception {
```

Comment:
if it is possible, could you also add test cases for deleting deleted blobs and deleting expired blobs?